fuzzer/cascade/memview.py

```diff
@@ -57,7 +57,8 @@ def get_available_contig_space(self, addr: int):
 
     # @param start:    first address of the range.
     # @param end:      last address of the range, excluded.
-    def alloc_mem_range(self, start: int, end: int):
+    def alloc_mem_range(self, start: int, alloc_range: int):
```

Comment:
I'd prefer `alloc_size` (or `size`) as `range` has a specific meaning in Python already.